pyclesperanto_prototype/_tier0/_push.py

```diff
@@ -3,13 +3,20 @@
 
 
 def push(any_array):
-    """Push numpy array on device to an OpenCL array on GPU.
+    """Copies an image specified by its name to GPU memory in order to process it there later.    Parameters
+    ----------
```

Comment:
Looks like your autogenerator is missing a couple of newlines? =)

```suggestion
    """Copies an image specified by its name to GPU memory in order to process it there later.
    
    Parameters
    ----------
```

Reply:
Also this docstring is missing a Returns section

Reply:
Drawbacks of auto-generating documentation. 😄